src/setup_node_env/ensure_node_preserve_symlinks.js

```diff
@@ -89,10 +89,18 @@
     }
 
     if (spawnResult.signal !== null) {
-      return 128 + spawnResult.signal;
+      console.log(
+        'ensure_node_preserve_symlinks wrapper: process exitted with signal',
+        spawnResult.signal
+      );
+      return 1;
```

Comment:
Should we still follow POSIX here by `return !isNaN(spawnResult.signal) ? 1 : 128 + spawnResult.signal;` ?

Reply:
The docs show the signal property is always a string so I don't think that's necessary. https://nodejs.org/dist/latest-v16.x/docs/api/child_process.html#child_processspawnsynccommand-args-options